files/12dSolutions/Georef_A_12d.ifc

```diff
@@ -0,0 +1,53 @@
+#14 = IFCLOCALPLACEMENT($, #15);
+#15 = IFCAXIS2PLACEMENT3D(#16, #17, #18);
+#16 = IFCCARTESIANPOINT((0., 0., 0.));
+#17 = IFCDIRECTION((0., 0., 1.));
+#18 = IFCDIRECTION((1., 0., 0.));
+#19 = IFCDIRECTION((0., 1.));
+#20 = IFCPROJECTEDCRS('EPSG:3857', $, 'World Geodetic System 1984 ensemble', $, $, $, #9);
+#21 = IFCMAPCONVERSIONSCALED(#13, #20, 16833194.0548, -4009644.0757, 0., 1., 0., 1., 1., 1., 1.);
+#22 = IFCRELDECLARES('0Nobr4I$PCH8WrE1DDD57X', #2, '', '', #1, (#23, #33, #34));
+#23 = IFCACTOR('1z$RasP$5Bdw_a6U0slDHi', $, $, $, $, #4);
+#24 = IFCCLASSIFICATION('Uniclass 2015', '1', '23-04-2018', 'Uniclass', 'Unified construction classification', 'uniclass.thenbs.com', $);
+#25 = IFCRELASSOCIATESCLASSIFICATION('1u_XjXELP6Gw0G_Cg212kI', #2, $, $, (#1), #24);
+#26 = IFCDOCUMENTINFORMATION('project_plan', 'project_plan.pdf', $, 'https://github.com/bSI-InfraRoom/IFC-Tunnel-Deployment/blob/main/project_plan.pdf', $, $, $, $, $, (#27), $, $, 'application/pdf', $, $, $, $);
+#27 = IFCORGANIZATION('bSI', 'Building Smart International', $, $, $);
+#28 = IFCRELASSOCIATESDOCUMENT('0XWfsOzTv04Pz$A58URhgy', #2, '', '', (#1), #26);
+#29 = IFCDATASETINFORMATION('Dataset-test', 'Dataset-test-xml', $, 'http://xxx.yyy.se/Datasets/Dataset-test.xml', $, $, $, $, $, $, $, $, 'text/xml', $, $, $, $, 'http://geosciml.org/geosciml/2.0/xsd/geosciml.xsd');
+#30 = IFCRELASSOCIATESDATASET('02ejFclqL42QLZHJpn8ANe', #2, '', '', (#1), #29);
+#31 = IFCLIBRARYINFORMATION('Example Library', '1.2.3', #3, '2023-03-10T13:05:38', 'http://xxx.yyy.se/Libraries/ExampleLibrary.ifc', $);
+#32 = IFCRELASSOCIATESLIBRARY('0bxjAKwz9DX9CrRGhBoFte', #2, '', '', (#1), #31);
+#33 = IFCARCHELEMENTTYPE('3l9$2hAxr6Hx$RitoS1PAI', #2, 'ArchType Example', $, $, $, $, $, 'MyArch', .USERDEFINED.);
+#34 = IFCPROPERTYSETTEMPLATE('30$TWUmhXCpQik7$iwa1Mi', #2, 'Pset_GeoObsBoreholeConstruction', $, .PSET_TYPEDRIVENOVERRIDE., 'IfcBorehole', (#35));
+#35 = IFCSIMPLEPROPERTYTEMPLATE('1lcv35HjrE1R$ee0lT6V6c', #2, 'DrillingDiameter', $, .P_SINGLEVALUE., 'IfcPositiveLengthMeasure', $, $, $, $, $, $);
+#36 = IFCSITE('0hf1$o_8H5dQQ1kuoHYlXh', #2, 'Default Site', 'Description of Default Site', $, #14, $, $, .ELEMENT., $, $, $, $, $);
+#37 = IFCRELAGGREGATES('1wUGpEYtn14gG33sBzRt7A', #2, 'ProjectContainer', 'ProjectContainer for Sites', #1, (#36));
+#38 = IFCFACILITY('09i1GJI4X3J8ubPZYfcSmx', #2, 'base', 'Description of base', $, #14, $, $, .ELEMENT.);
+#39 = IFCRELAGGREGATES('2V4VdApF1BdPSvMLZkah4J', #2, 'SiteContainer', 'SiteContainer For All Models', #36, (#38));
+#40 = IFCRELCONTAINEDINSPATIALSTRUCTURE('2EXlqCgUPDjO6fvIKZmHrY', #2, 'FacilityContainer', 'FacilityContainer for Elements', (#41), #38);
+#41 = IFCBUILDINGELEMENTPROXY('2wTV1IXaf4Ev6UbKzFLN7o', #2, 'Super', $, 'Super', #14, #42, $, .NOTDEFINED.);
+#42 = IFCPRODUCTDEFINITIONSHAPE($, $, (#43));
+#43 = IFCSHAPEREPRESENTATION(#13, 'Body', 'Point', (#44));
+#44 = IFCCARTESIANPOINT((-16833184.0548, 4009649.0757, 0.));
```

Comment:
This point seems to negate almost the whole offset of the `IfcMapConversionScaled`, however in engineering coordinates (not mapped). Is this intentional?

Reply:
I was just using a random point for the purposes of getting our software to export a file, so the offset defined in IfcMapConversionScaled will be mostly negated as I just put a point close to the (0,0,0) before I setup the map conversion.